### ISSUE TITLE
Kubernetes v2: handle ServiceAccount kind as part of manifest file

### ISSUE THREAD

**Issue by danilina-wsib** [spinnaker/spinnaker#2660]
v1.6.1, K8s v2 provider, Azure AKS

I put a ServiceAccount manifest in the same file as my deployment

I get NullPointerException at com.netflix.spinnaker.clouddriver.kubernetes.v2.op.manifest.KubernetesDeployManifestOperation.operation, line 112

After I looked at the code, I don;t see ServiceAccount handler defined, looks like Kubernetes v2 provider is not able to deploy ServiceAccount manifests

**Comment by lwander** [comment]
Agreed, should be supported. In the mean time, you can register your own service account handler using:

```yaml
providers:
  kubernetes:
    accounts:
    - name: my-k8s-account
      customResources:
      - kind: ServiceAccount
      # other account properties...
```

in your halconfig, or clouddriver.yml

**Comment by lwander** [comment]
This works for CRDs/TPRs as well

**Comment by lwander** [comment]
Looks like it already is supported, see [here](https://github.com/spinnaker/clouddriver/blob/master/clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/v2/op/handler/KubernetesServiceAccountHandler.java). Will be released in 1.7 this week.
